### DISCUSSION TITLE
Documentation example with custom config flag does not work

### DISCUSSION THREAD

**Discussion by graforlock** [docker-library/mongo#586] [General]
For example, when you try:

`docker run --name some-mongo -d mongo --config /etc/mongod.conf.orig`

Which is re-using existing configuration, then you are getting exit code 100 and the container doesn't start.

**Comment by tianon** [comment]
This is a problem with the specific contents of `/etc/mongod.conf.orig` (not the example in the image description, which doesn't reference that file).

Digging into it interactively, I see that config file uses `/var/log/mongodb/mongod.log` for logging -- after attempting to start using that config, I checked that log file and found (not unexpectedly): `{"t":{"$date":"2022-11-28T23:08:24.988+00:00"},"s":"E",  "c":"CONTROL",  "id":20557,   "ctx":"initandlisten","msg":"DBException in initAndListen, terminating","attr":{"error":"NonExistentPath: Data directory /var/lib/mongodb not found. Create the missing directory or specify another path using (1) the --dbpath command line option, or (2) by adding the 'storage.dbPath' option in the configuration file."}}`

(We don't configure/use `/var/lib/mongodb` in this image, but rather use the built-in default `/data/db` of `mongod` itself.)
